packages/core/src/components/buttons/buttons.module.scss

```diff
@@ -9,54 +9,47 @@
 /* 'primary' | 'secondary' | 'link' | 'outlined'; */
 
 .appearance-primary {
-	color: white;
-	background: $colors-primary-a2 !important;
+	color: $white;
 	border: none;
 	border-radius: 8px;
 
 	&:hover {
-		background: $colors-accents-2 !important;
+		background: $colors-accents-2;
 	}
 
 	&:focus {
-		background: $colors-warning-1!important;
+		background: $colors-warning-1;
 		color: $colors-neutral-8;
 		box-shadow: 0px 2px 0px $colors-neutral-8;
+		outline: none;
 	}
-}
-
-.appearance-primary:disabled {
-	cursor: not-allowed !important;
-	background: $colors-primary-a1;
 
-	&:hover {
-		background: $colors-neutral-6 !important;
+	&:disabled {
+		cursor: not-allowed !important;
+		background: $colors-neutral-6;
```

Comment:
shouldn't this color be `colors-primary-a1` ? same for disabled->hover?

Reply:
yep